<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>379</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>337</width>
    <height>379</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YARR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>39</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>39</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>337</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionSave_Game"/>
    <addaction name="actionLoad_Game"/>
    <addaction name="actionResign"/>
    <addaction name="actionNetwork_Game"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>&amp;Preferences</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionRules"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_YARR"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRules">
   <property name="text">
    <string>&amp;Rules</string>
   </property>
  </action>
  <action name="actionAbout_YARR">
   <property name="text">
    <string>&amp;About YARR</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;New Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionResign">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Re&amp;sign</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave_Game">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_Game">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionNetwork_Game">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Network Game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
